www/content/attributes/hx-headers.md

```diff
@@ -7,10 +7,14 @@ The `hx-headers` attribute allows you to add to the headers that will be submitt
 By default, the value of this attribute is a list of name-expression values in [JSON (JavaScript Object Notation)](https://www.json.org/json-en.html) 
 format.
 
+```html
+  <div hx-get="/example" hx-headers='{"myHeader": "My Value"}'>Get Some HTML, Including A Custom Header in the Request</div>
+```
+
 If you wish for `hx-headers` to *evaluate* the values given, you can prefix the values with `javascript:` or `js:`.
 
 ```html
-  <div hx-get="/example" hx-headers='{"myHeader": "My Value"}'>Get Some HTML, Including A Custom Header in the Request</div>
+  <body hx-headers='js:{ "Locale" : new Intl.DateTimeFormat().resolvedOptions().locale }'>...</body>
```

Comment:
I think to keep the example consistent, we could take inspiration from the [`hx-vals`](https://htmx.org/attributes/hx-vals/) documentation, which has the following example:

```html
<div hx-get="/example" hx-vals='{"myVal": "My Value"}'>Get Some HTML, Including A Value in the Request</div>

<div hx-get="/example" hx-vals='js:{myVal: calculateValue()}'>Get Some HTML, Including a Dynamic Value from Javascript in the Request</div>
```

I think it's a concise & effective approach, using the same key for the same context element, just changing _how_ the value is retrieved and illustrating how `js:` works.

Let me know your thoughts on this! Your opinion will have more value than mine on this as I'm likely biased by my htmx experience (as you said, we want the docs to be clear enough for newcomers!)